model/riscv_extensions.sail

```diff
@@ -399,8 +402,32 @@ let extensions_ordered_for_isa_string = [
   // S and U are not valid extensions in the device-tree, whereas H is.
 
   // Z extensions, ordered by category and then alphabetically.
+  // Za
+  Ext_Zaamo,
+  Ext_Zabha,
+  Ext_Zacas,
+  Ext_Zalrsc,
+  Ext_Zawrs,
+
+  // Zb
+  Ext_Zba,
+  Ext_Zbb,
+  Ext_Zbc,
+  Ext_Zbkb,
+  Ext_Zbkc,
+  Ext_Zbkx,
+  Ext_Zbs,
+
+  // Zc
+  Ext_Zca,
+  Ext_Zcb,
+  Ext_Zcd,
+  Ext_Zcf,
+  Ext_Zcmop,
+
```

Comment:
As the top comment says, this list should be ordered by category and then alphabetically _within a category_.  The categories themselves are not ordered alphabetically.   This list was ordered correctly the way it was.  See the "Subset Naming Convention" in the unpriv manual.